.github/ISSUE_TEMPLATE/BUG_REPORT.yaml

```diff
@@ -98,3 +111,9 @@ body:
   attributes:
     label: Additional Context
     description: Please add any other infos that could be useful.
+- type: markdown
+  attributes:
+    value: |
+      ## Vote for this issue
+      If you come accross this issue and would like to see it taken care of, please vote with a thumbs up 👍.
+      If you have any additional thoughts or use cases, please leave a comment.
```

Comment:
I think upvoting is a normal process on opensource issues not sure if we need to mention that.